x-pack/test/cloud_security_posture_functional/page_objects/findings_page.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { FtrProviderContext } from '../ftr_provider_context';
+
+const FINDINGS_ROUTE = 'cloud_security_posture/findings';
+const SECURITY_SOLUTION_APP_NAME = 'securitySolution';
+const EUI_PAGINATION_BUTTON = 'tablePaginationPopoverButton';
+const FINDINGS_NAME_COLUMN_TESTID = 'tableHeaderCell_rule.name_5';
```

Comment:
these are duplicated from our plugin, which seems common in FTR 

we might want to: 
1 - extract these to `@kbn/cloud-security-posture-test-subjects` and re-use in our plugin/ftr tests.
2 - export from our plugin's common folder 
3 - export from our plugin's public folder a type (and still duplicate, but safely)